architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,194 @@
+a policy], showing rule sets for the *admin* and *basic-user*
+link:#roles[default roles]:
+
+====
+
+[options="nowrap"]
+----
+admin			Verbs					Resources															Resource Names	Extension
+			[create delete get list update watch]	[projects resourcegroup:exposedkube resourcegroup:exposedopenshift resourcegroup:granter secrets]				[]
+			[get list watch]			[resourcegroup:allkube resourcegroup:allkube-status resourcegroup:allopenshift-status resourcegroup:policy]			[]
+basic-user		Verbs					Resources															Resource Names	Extension
+			[get]					[users]																[~]
+			[list]					[projectrequests]														[]
+			[list]					[projects]															[]
+			[create]				[subjectaccessreviews]														[]		IsPersonalSubjectAccessReview
+----
+====
+
+The following excerpt from
+link:../../admin_guide/manage_authorization_policy.html#viewing-roles-and-bindings[viewing
+policy bindings] shows the above roles bound to various users and groups:
+
+====
+
+[options="nowrap"]
+----
+RoleBinding[admins]:
+				Role:	admin
+				Users:	[alice system:admin]
+				Groups:	[]
+RoleBinding[basic-user]:
+				Role:	basic-user
+				Users:	[joe]
+				Groups:	[devel]
+----
+====
+
+== Evaluating Authorization
+
+Several factors are combined to make the decision when OpenShift evaluates
+authorization:
+
+[cols="1,7"]
+|===
+
+|[[identity]]*Identity* |Both the user name and list of groups the user belongs to.
+
+|[[action]]*Action* a|The action being performed. In most cases, this consists of:
+
+[horizontal]
+Project:: The link:../../dev_guide/projects.html[project] being accessed.
+Verb:: Can be `get`, `list`, `create`, `update`, or others.
+Resource:: The API endpoint being accessed.
+
+|*Bindings* |The full list of link:#bindings[bindings].
+
+|===
+
+OpenShift evaluates authorizations using the following steps:
+
+. The identity and the project-scoped action is used to find all bindings that
+apply to the user or their groups.
+. Bindings are used to locate all the roles that apply.
+. Roles are used to find all the rules that apply.
+. The action is checked against each rule to find a match.
+. If no matching rule is found, the action is then denied by default.
+
+== Cluster Policy and Local Policy
+There are two levels of authorization policy:
+
+[cols="1,4"]
+|===
+
+|*Cluster policy* |link:#roles[Roles] and bindings that are applicable across
+all projects. Roles that exist in the cluster policy are considered _cluster
+roles_. Cluster bindings can only reference cluster roles.
+
+|*Local policy* |link:#roles[Roles] and bindings that are scoped to a given
+project. Roles that exist only in a local policy are considered _local roles_.
+Local bindings can reference both cluster and local roles.
+
+|===
+
+This two-level hierarchy allows re-usability over multiple projects with cluster
+policy while allowing controlled customization inside of individual projects
+with local policy.
```

Comment:
I think this sentence can be taken two ways:

"This two-level hierarchy allows re-usability over _any_ projects _that have_ cluster policy, while allowing controlled customization inside of individual projects _that have a_ local policy."

or

"This two-level hierarchy allows re-usability over multiple projects _by using a_ cluster policy, while allowing controlled customization inside of individual projects _by using a_ local policy."

Thoughts?

I'm also thinking "controlled customization" is redundant. Isn't all customization controlled? Suggesting it just be "customization".